code/game/objects/structures/crates_lockers/closets/secure/hydroponics.dm

```diff
@@ -11,3 +11,75 @@
 	new /obj/item/cultivator(src)
 	new /obj/item/hatchet(src)
 	new /obj/item/secateurs(src)
+
+//Plocker
+/obj/structure/closet/secure_closet/peach
+	name = "peach locker"
+	desc = "A huge hollow pitless peach."
+	locked = FALSE
+	icon_state = "plocker"
+	max_integrity = 40
+	armor = list(MELEE = 10, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 60, FIRE = 10, ACID = 0)
+	can_weld_shut = FALSE
+	damage_deflection = 0
+	material_drop_amount = 0
+	close_sound = "sound/misc/moist_impact.ogg"
+	open_sound = "sound/misc/soggy.ogg"
+	breakout_time = 100
+	var/isDegrading = FALSE //Used to check if the peach locker is degrading
+	var/owner = null //The owner of the locker which is set when an impeach is activated
+	var/lifespan = 3 MINUTES //Right when the peach locker is created it has this amount of time before it starts degrading
```

Comment:
You have generated new variables/procs/defines without proper documentation for the autodoc to work.

Please click [here ](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)for a guide on how to properly document these items.